tracer/src/Datadog.Trace/Logging/DirectSubmission/Formatting/LogFormatter.cs

```diff
@@ -229,6 +230,12 @@ internal static JsonTextWriter GetJsonWriter(StringBuilder builder)
                 writer.WriteValue(exception.ToString());
             }
 
+            if (!string.IsNullOrEmpty(logCategory))
+            {
+                WritePropertyName(writer, "SourceContext");
```

Comment:
To answer your open question, I'd say the likelihood of conflicting with a SourceContext property is close to nil. But I assume it wouldn't be more way complicated to name it `DatadogSourcecontext`, which would reduce completely the risk. WDYT?

Reply:
We could. But then again we pay for every byte... Also, it would mean we have to check for `SourceContext` when iterating the Serilog properties, and duplicate the field as `DatadogSourceContext` (renaming it would be a breaking change). So IMO it's not worth the hassle 😅 But I can be convinced if anyone (including you) is passionate about it 😉

Reply:
You know I wasn't passionate in my first message ;). The "we pay for every byte" isn't relevant. We could even send less bytes on 3 loggers :p But, anyway, as you say that may not be worth the extra hassle. 

The other option is to properly document it. To at least hopefully avoid support questions if we get the issue one day